components/EmotionForm.js

```diff
@@ -68,95 +68,98 @@ export default function EmotionForm({
 
     await onSubmit(inputData);
     event.target.reset();
+    showToastMessage("Emotion successfully submitted!");
   }
 
   if (isLoading) {
     return <h1>Loading...</h1>;
   }
 
+  if (emotionTypes?.length === 0) {
+    return <h1>Keine Emotionsarten verfügbar</h1>;
+  }
+
   return (
-    <>
-      <StyledFormContainer>
-        <StyledFormHead onClick={toggleVisibilityForm}>
-          <StyledSubheadline>
-            {editMode ? "Update your Emotion" : "Add your Emotion"}
-          </StyledSubheadline>
-          {!editMode && (
-            <StyledVisibilityIcons aria-label="show-hide-form">
-              {formVisibility ? <MinusIcon /> : <PlusIcon />}
-            </StyledVisibilityIcons>
+    <StyledFormContainer>
+      <StyledFormHead onClick={toggleVisibilityForm}>
+        <StyledSubheadline>
+          {editMode ? "Update your Emotion" : "Add your Emotion"}
+        </StyledSubheadline>
+        {!editMode && (
+          <StyledVisibilityIcons aria-label="show-hide-form">
+            {formVisibility ? <MinusIcon /> : <PlusIcon />}
+          </StyledVisibilityIcons>
+        )}
+      </StyledFormHead>
+
+      <StyledEmotionForm $isVisible={formVisibility} onSubmit={handleSubmit}>
+        <label htmlFor="type">Emotion (type)*</label>
+        <SelectEmotionContainer>
+          <StyledSelectEmotion
+            value={selectedEmotionType}
+            id="type"
+            name="type"
+            onChange={handleChangeEmotionType}
+          >
+            <option value="">---Choose an Emotion---</option>
+            {emotionTypes.map((emotion) => (
+              <option key={emotion._id} value={emotion._id}>
+                {emotion.name}
+              </option>
+            ))}
+          </StyledSelectEmotion>
+          <StyledArrow>▼</StyledArrow>
+        </SelectEmotionContainer>
+        {!editMode && (
+          <StyledCreateEmotionLink
+            href="/createemotiontype"
+            aria-label="Create a new emotion type"
+          >
+            Create your Emotion Type <MoodPlusIcon />
+          </StyledCreateEmotionLink>
+        )}
+        <label htmlFor="intensity">Emotion intensity*</label>
+        <SliderIntensity
```

Comment:
How do you decide where to add an aria label? For example, you have a aria label for creating a new emotion type but not for this slider.